src/jukugo_game/game/forms.py

```diff
@@ -1,10 +1,39 @@
+from typing import Tuple
+
 from django import forms
 
-from .models import Jukugo
+from .models import Computer, Player
 
 
 class JukugoForm(forms.ModelForm):
     class Meta:
-        model = Jukugo
+        model = Player
         app_label = "game"
         fields = ("jukugo",)
+
+
+class LevelChoiceForm(forms.Form):
```

Comment:
Is a choice form correct for this function?

Reply:
Let me say my question in a different way.
Is nothing better than this form for this function?

Reply:
It may be better to display all modes initially than pull-down formation.